R/AllGenerics.R

```diff
@@ -230,17 +230,18 @@ setReplaceMethod("nonSplicedReads", "FraserDataSet", function(object, value){
 #' @param x A \code{FraserDataSet} object
 #' @param i A integer vector to subset the rows/ranges
 #' @param j A integer vector to subset the columns/samples
-#' @param by a character (j or ss) definig if we subset by
+#' @param by a character (j or ss) defining if we subset by
 #'             junctions or splice sites
+#' @param drop Currently not used 
```

Comment:
did you run `devtools::document()` i think this did not populate into the documentation.